mod-pubsub-server/src/main/java/org/folio/services/impl/SecurityManagerImpl.java

```diff
@@ -206,71 +196,60 @@ public SecurityManagerImpl(@Autowired Vertx vertx, @Autowired Cache cache,
       });
   }
 
-  private Future<Boolean> assignPermissions(String userId, OkapiConnectionParams params) {
-    List<String> permissions = readPermissionsFromResource(PERMISSIONS_FILE_PATH);
-    if (CollectionUtils.isEmpty(permissions)) {
-      LOGGER.info("No permissions found to assign to {} user", systemUserConfig.getName());
-      return Future.succeededFuture(false);
-    }
+  private Future<Void> assignPermissions(String userId, OkapiConnectionParams params) {
     JsonObject requestBody = new JsonObject()
       .put("id", UUID.randomUUID().toString())
       .put("userId", userId)
-      .put("permissions", new JsonArray(permissions));
+      .put("permissions", new JsonArray(PERMISSIONS));
     return doRequest(params, PERMISSIONS_URL, HttpMethod.POST, requestBody.encode())
       .compose(response -> {
-        Promise<Boolean> promise = Promise.promise();
         if (response.getCode() == HttpStatus.HTTP_CREATED.toInt()) {
-          LOGGER.info("Added permissions [{}] for {} user", StringUtils.join(permissions, ","),
+          LOGGER.info("Added permissions [{}] for {} user", StringUtils.join(PERMISSIONS, ","),
             systemUserConfig.getName());
-          promise.complete(true);
+          return Future.succeededFuture();
         } else {
```

Comment:
I think mod-pubsub needs to ignore status 400 here.. Perhaps mod-pubsub adds permissions that are redundant and mod-permissions returns that error.. But current platform builds fail because if it.

Reply:
You are right, but this is out of scope of this PR. This PR is a code cleanup without changing behavior.
Please create a separate Jira and PR.